src/control/ballHandling/DribbleBackwards.cpp

```diff
@@ -218,12 +218,6 @@ RobotCommand DribbleBackwards::sendDribbleBackwardsCommand() {
     interface::Input::drawData(interface::Visual::BALL_HANDLING, {backwardsDribbleLine.first, backwardsDribbleLine.second},
             Qt::white, robot->id, interface::Drawing::LINES_CONNECTED);
 
-    // check if the ball is not too far right or too far left of the robot, and try to compensate for that
-    if (ball->visible && false) {
-        Angle ballAngleRelativeToRobot = (ball->pos - robot->pos).toAngle() - robot->angle;
-        command.vel += (robot->angle + M_PI_2).toVector2(ballAngleRelativeToRobot);
-    }
-
```

Comment:
Same thing, discussed this with thijs? I do not know the reason for this code, or why you removed it.